charm/ksql/config.yaml

```diff
@@ -0,0 +1,14 @@
+options:
```

Comment:
Need to remove these example options. If there aren't any options, need to declare it as an empty yaml map. That's not explained very well (if at all) in the charmer documentation.

Reply:
Or... maybe there are useful config options we could add here? Like log level, perhaps?

Reply:
@cmars I think we should put this in a separate PR, so we don't create a fat PR